pmd-core/src/main/java/net/sourceforge/pmd/cli/PMDParameters.java

```diff
@@ -121,6 +125,17 @@ public class PMDParameters {
                    + "If this option is not specified, the report is rendered to standard output.")
     private String reportfile = null;
 
+    @Parameter(names = { RELATIVIZE_PATHS_WITH, "-z" },
+               variableArity = true,
+               description = "Path relative to which directories are rendered in the report."
+                             + "This option allows shortening directories in the report; "
+                             + "without it, paths are rendered as absolute paths. "
+                             + "The option can be repeated, in which case the shortest relative path will be used."
+                             + "This option replaces --short-names since PMD 6.54.0.",
```

Comment:
I'm unsure about having this here…

We are bound to forget removing it even after it stops making sense, the doc already states that, and at runtime we already have the suggested replacements being output as warning.

Reply:
You mean the last sentence "This option replaces --short-names since PMD 6.54.0."?

Hm...yes, it would be now the only case, where we output it in the parameter help. We have it documented and we issue a deprecation warning, when "--short-names" is still used.

Yes, I guess, it's safe to remove this last sentence here.

Reply:
I've removed this sentence now.